sys/dev/etherswitch/e6000sw/e6000sw.c

```diff
@@ -260,8 +260,9 @@ e6000sw_probe(device_t dev)
 		 */
 		resource_int_value(device_get_name(sc->dev),
 		    device_get_unit(sc->dev), "is8190", &is_6190);
-	resource_int_value(device_get_name(sc->dev),
-		device_get_unit(sc->dev), "is6190x", &is_6190x);
+	if (resource_int_value(device_get_name(sc->dev),
+		device_get_unit(sc->dev), "is6190x", &is_6190x) != 0)
+		return;
```

Comment:
THis should avoids the following error: `error: misleading indentation; statement is not part of the previous 'if' [-Werror,-Wmisleading-indentation]`